### DISCUSSION TITLE
Joomla! 3.9.26 Release Candidate

### DISCUSSION THREAD

**Discussion by HLeithner** [joomla/joomla-cms#33059] [General]
New Joomla! Installations [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Full_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Full_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Full_Package.zip)
Update from Joomla! 3.9.25 [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.25_to_3.9.26-rc-Release_Candidate-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.25_to_3.9.26-rc-Release_Candidate-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.25_to_3.9.26-rc-Release_Candidate-Patch_Package.zip)
Update from Joomla! 3.9.x [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.x_to_3.9.26-rc-Release_Candidate-Patch_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.x_to_3.9.26-rc-Release_Candidate-Patch_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.x_to_3.9.26-rc-Release_Candidate-Patch_Package.zip)
Update from Joomla! 2.5 or previous 3.x releases [.tar.bz2](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Update_Package.tar.bz2) | [.tar.gz](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Update_Package.tar.gz) | [.zip](https://github.com/joomla/joomla-cms/releases/download/3.9.26-rc/Joomla_3.9.26-rc-Release_Candidate-Update_Package.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/joomla/joomla-cms/releases/tag/3.9.26-rc'>Joomla! 3.9.26 Release Candidate</a>.</em>

**Comment by PhilETaylor** [comment]
Oh this is a nice new GitHub Feature - auto discussions on releases.... FIRST! /called it/ :-)

**Comment by HLeithner** [comment]
@nikosdion in this release are bugfixes for 2 of your reported issues (https://github.com/joomla/joomla-cms/pull/32862 and https://github.com/joomla/joomla-cms/pull/33040) since you created a workaround it would be great if you could test the RC and give feedback if this doesn't breaks your workaround and fix your reported issues

thanks

**Reply by joomdonation** [reply]
Base on what he wrote on his website, afraid of he is not notified anymore when he is mentioned from our repo. Maybe @PhilETaylor can ask him directly (guess you have direct contact with him)
![Working-around-Joomla’s-broken-extensions-updater-Akeeba-Ltd](https://user-images.githubusercontent.com/977664/113969535-5d537e00-985f-11eb-843e-101065d509c3.png)

**Reply by nikosdion** [reply]
I am indeed blocking the emails since GitHub doesn't have an option "Unsubscribe me from this thread and never subscribe me again". It's more peaceful now. I like it.

Regarding the two bug fixes, they do what I am already doing in my workaround code — there are only so many ways to clear a cache, the same one we both use. That is to say, we're both cleaning the opcode cache for the installed files and clearing the pesky `_system` cache. With 3.9.26 this code essentially runs twice, once in Joomla itself and once in my post-installation script.

Since my software can be and _is still_ used on older Joomla 3.9 releases I can't remove my workaround. I will only be able to remove my workaround when I completely drop support for Joomla 3.9 about 3 months after 3.10 stable is out.

I'd like to thank @PhilETaylor for submitting the PRs to these bug fixes and you guys for testing and merging them. These small issues might seem trivial but contribute to what can best be described as "death by a thousand paper cuts" if left unpatched, especially if they have been around for a decade. There are plenty more. Hopefully we'll see them addressed. Last time this happened, in 2014, it marked the best sales years we've ever had. People care about the small things.

**Reply by HLeithner** [reply]
Thanks for the confirmation.
